jpm/package.json

```diff
@@ -9,7 +9,7 @@
   "permissions": {
     "private-browsing": true
   },
-  "main": "index.js",
+  "main": "main.js",
```

Comment:
@chadxz Heads up, I took liberty to rename the jpm entry point here to main.js for consistency, but everything else was needed to get the extension to run in Firefox at all. It's really just a handful of tweaks, but it looks double because of support for both cfx and jpm extension types.